frontend/src/ClassDetail.svelte

```diff
@@ -60,7 +60,7 @@
 
       for (let i = 0; i < clazz.students.length; i++) {
           const student = clazz.students[i];
-          if (!isNaN(clazz.assignments[assignment_index].students[student.username].assigned_points)) {
+          if (!isNaN(parseFloat(clazz.assignments[assignment_index].students[student.username].assigned_points))) {
```

Comment:
This is a bit hacky. Could we do something like this instead?
```suggestion
          const points = clazz.assignments[assignment_index].students[student.username].assigned_points;
          if (!isNaN(points) && points !== null) {
```

Reply:
Of course we could :) I was just afraid of creating a variable for that.